plugins/src/core/handlers.ts

```diff
@@ -0,0 +1,17 @@
+import { PushNotification } from "./push";
+import { DeepLink } from "./ddl";
+export interface PushNotificationHandler {
+  (notification: PushNotification): void;
+}
+
+export interface InAppDeepLinkHandler {
+  (data: { [key: string]: any }): void;
```

Comment:
InAppButtonPress is the argument, which is 
```
public struct InAppButtonPress {
    public let deepLinkData : [AnyHashable:Any]
    public let messageId : Int64
    public let messageData : NSDictionary?

    init(deepLinkData:[AnyHashable:Any], messageId:Int64, messageData:NSDictionary?) {
        self.deepLinkData = deepLinkData
        self.messageId = messageId
        self.messageData = messageData
    }
}
```

in my test i've received this `{"messageData":null,"deepLinkData":{"hellow":"win"},"messageId":119}`